book/src/examples.md

```diff
@@ -4,7 +4,7 @@
 * [S2 example](https://github.com/google/autocxx/tree/main/examples/s2) - example using S2 geometry library
 * [Steam example](https://github.com/google/autocxx/tree/main/examples/steam-mini) - example using (something like) the Steam client library
 * [Subclass example](https://github.com/google/autocxx/tree/main/examples/subclass) - example using subclasses
-* [Integration tests](https://github.com/google/autocxx/blob/main/integration-tests/src/tests.rs)
+* [Integration tests](https://github.com/google/autocxx/blob/main/integration-tests/src/lib.rs)
```

Comment:
Thanks very much, good spot! However this is the file we now need to link to:
https://github.com/google/autocxx/blob/main/integration-tests/tests/integration_test.rs

Reply:
> Thanks very much, good spot! However this is the file we now need to link to: https://github.com/google/autocxx/blob/main/integration-tests/tests/integration_test.rs

Thank you. Fixed